Recrutify/Recrutify.DataAccess/Repositories/Abstract/IScheduleRepository.cs

```diff
@@ -9,6 +9,8 @@ public interface IScheduleRepository
     {
         Task<List<Schedule>> GetByUserPrimarySkillAsync(IEnumerable<Guid> userIds,  DateTime date, Guid primarySkillId);
 
+        Task UpdateScheduleSlotsCandidateInfoAsync(IEnumerable<InterviewAppointment> appointInterviews);
```

Comment:
interviewAppointments, and name of param in interface should be same as in class